build.assets/makefiles/master/k8s-master/cluster-dns.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Prior to upgrade to Kubernetes 1.19 the list of cluster DNS servers are provided
+# through the kubelet flag `--cluster-dns`. This flag is now deprecated and the cluster
+# DNS servers must be specified via the kubelet config file under `clusterDNS`.
+#
+# This script writes the cluster DNS servers into the kubelet config file.
+#
+# More info on `--cluster-dns` can be found at: https://v1-19.docs.kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
+
+# Environment file that contains the cluster DNS_ADDRESSES.
+DNS_ENV=/run/dns.env
+# Path to kubelet config file.
+KUBELET_CONFIG=/etc/kubernetes/kubelet.yaml
+
+# If /run/dns.env file does not exist exit without any changes.
+if [ ! -f $DNS_ENV ]
+then
+    echo "$DNS_ENV does not exist"
+    exit
+fi
+
+# Import DNS_ADDRESSES from env file.
+export $(cat $DNS_ENV | xargs)
+
+# If DNS_ADDRESSES is empty exit without any changes.
+if [ -z "$DNS_ADDRESSES" ]
+then
+    echo "DNS_ADDRESSES is not set"
+    exit
+fi
+
+# Remove existing clusterDNS values.
+perl -i -p0e 's/clusterDNS:(\n  - .*)*\n//g' $KUBELET_CONFIG
```

Comment:
I know it is up to you which tool to use here, but `sed`, in my opinion, is better known by anyone who will support the code and can provide the same result.

Reply:
Thats fair, I've updated the script in 6e2b26f to use `sed` instead of `perl`. I'm not too familiar with `sed`, so let me know if the command can be simplified.